### DISCUSSION TITLE
I can't access my account due to Authy messing up my codes and I want to find a way of verifying my identity and recovering it

### DISCUSSION THREAD

**Discussion by ffyeins** [community/community#145432] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I want to clarify I am not writing this post from the account in question.

Some days ago I had to format my phone due to some issues and lost all my 2FA codes on my Authy account. I took the required precautions needed for backing up my codes but I still lost them all. I've been trying to recover them for a week so far but with no luck.

I tried using the GitHub assistant to reset the 2FA and gain access to my account but it seems that with no recovery codes it is impossible. I hit the wall that tells me that there is nothing else that can be done and that my account is lost forever.

I understand the security measures and that I should've been more responsible with my recovery codes, but I'm frustrated because I am put in this situation because of Authy's fault, not mine.

I want to know if there's any way of recovering my account by verifying my identity and the devices that have had access to my account. I have a lot of means of proving I am the owner of the account:

- My username is my real name+surname.
- The email used for the account is also my real name+surname and I still have access to it.
- I still own three devices that have had access to that account throughout the last years and one even throughout the last decade: my personal PC, my personal MacBook, and my phone that had the Android app installed.

I want to know if there is a chance of recovering my account by providing documents and photos or videos of me that verify my identity.

**Comment by davevad93** [comment]
Hi @ffyeins ,

The best way to handle account related problems is by contacting support [here](https://support.github.com/contact/account), under **Account** select `Other`, then select `Two-factor authentication recovery` and explain your issue with your main GitHub account.

**Reply by ffyeins** [reply]
Thank you. I didn't know I had this option available. I hit the wall with the virtual assistance and couldn't figure out my next option.

**Reply by davevad93** [reply]
You're welcome @ffyeins , I hope they can assist you and solve it as soon as possible! 👍

**Reply by ffyeins** [reply]
Hey @davevad93, I have tried contacting support but my ticket has been open for 2 weeks now with no replies so far. Do you know if this is normal? Should I do something else about it?

**Reply by davevad93** [reply]
Hey @ffyeins yes, support can take a while to reply because they have a lot of daily requests to deal with, I suggest you to be patient because it can take even 1 month or more; nevertheless I hope they will answer you asap!

**Reply by ffyeins** [reply]
I'll be patient then. Thanks!
